packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -180,7 +188,13 @@ export function getAllTaggedElements({
   for (const { node, text } of stegaNodes) {
     const sourceMap = decode(text);
     if (!sourceMap || !sourceMap.origin.includes('contentful.com')) {
-      debug.warn;
+      debug.warn(
+        "Element has missing or invalid ContentSourceMap, please check if you have correctly enabled ContentSourceMaps and that the element's data originates from Contentful",
```

Comment:
are we trying to imply that the data shouldn't be transformed once it comes from our APIs?

Reply:
I have to admit I'm not as familiar with CSM so that was my understanding, but if that's an inaccurate description please let me know what would be more accurate

Reply:
@2wce @YvesRijckaert Can you confirm if this is an appropriate message?